script.js

```diff
@@ -0,0 +1,73 @@
+// Collection of books
+let books = [];
+
+// Load books from local storage
+const loadBooks = () => {
+  const storedBooks = JSON.parse(localStorage.getItem('books'));
+  if (storedBooks) {
+    /* eslint-disable no-const-assign */
+    /* eslint-disable no-undef */
+    bookCollection = storedBooks;
+  }
```

Comment:
- [ ] Kindly make sure to collect the name of the variable where all books are being restored from local storage in order to be rendered on the page once the page loads.  At this moment we can not load books from local storage because you mistakenly changed the name to `bookCollection` instead of `books`. Kindly fix that.